src/main/drivers/vtx_rtc6705.c

```diff
@@ -110,7 +110,9 @@ bool rtc6705IOInit(const vtxIOConfig_t *vtxIOConfig)
         IOConfigGPIO(vtxPowerPin, IOCFG_OUT_PP);
     }
 
-    if (vtxIOConfig->csTag && spiSetBusInstance(dev, vtxIOConfig->spiDevice)) {
+    // RTC6705 when using SOFT SPI driver doesn't use an SPI device, so don't attempt to initialise an spiInstance.
+    SPI_TypeDef *spiInstance = spiInstanceByDevice(SPI_CFG_TO_DEV(vtxIOConfig->spiDevice));
+    if (spiInstance && spiSetBusInstance(dev, vtxIOConfig->spiDevice)) {
```

Comment:
I think we should be checking for an SPI CS pin being configured here as well.

Reply:
See line 99, it's already done.